src/main.ts

```diff
@@ -771,8 +771,35 @@ export class Purchases {
    * @public
    */
   public static generateRevenueCatAnonymousAppUserId(): string {
-    const uuid = crypto.randomUUID();
-    return `${ANONYMOUS_PREFIX}${uuid.replace(/-/g, "")}`;
+    const generateUUID = (): string => {
```

Comment:
might be worth extracting into a utils file?

Reply:
Done